### PR TITLE
Cloudwatch: Fix AWS WAF and AWS DDoSProtection metrics

### LINKED ISSUE #24499: CloudWatch/Metrics: AWS Shield Support, dimensions not available
hello
I am using AWS shield advanced service.
 The shield service is "AWS / DDoSProtection" in the
 grafana: {"AllowedRequests", "BlockedRequests", "CountedRequests", "DDoSAttackBitsPerSecond", "DDoSAttackPacketsPerSecond", "DDoSAttackRequestsPerSecond", "DDoSDetected", "PassedRequests" 

But Can't load shield protected resource values.

'shield advanced' and 'AWS/DDoSprotection in Grafana' different services?
image

### PR DESCRIPTION
**Pull request by papagian**
**What this PR does / why we need it**:
This PR defines `AWS WAF` metrics and dimensions separately from `AWS Shield Advanced`  as described in [this](https://docs.aws.amazon.com/waf/latest/developerguide/monitoring-cloudwatch.html) document.

**Which issue(s) this PR fixes**:

Fixes #24499

**Special notes for your reviewer**:
- Non custom metrics are expected to [start with `AWS/`](https://github.com/grafana/grafana/blob/3a469f6ca25fa7bb56c58f9921dfe02e82df4d26/pkg/tsdb/cloudwatch/metric_find_query.go#L853)

- `AWSDDOSProtectionMetrics` are available only to Shield Advanced customers

### CODE REVIEW

**Comment by papagian** [comment]
@JuH0LEE

how do you run grafana? are you able to check if this fixes your Cloudwatch issue?

**Review by marefr — APPROVED** [review]
LGTM

**Review by JuH0LEE — COMMENTED** [review]
@papagian hi.
This part seems to have changed the details of the attackvector. (*ACKFlood, ChargenReflection...etc) In my opinion the dimesions are correct changes to "AWS / DDoSProtection": {"attackvector", "resourcearn"},

File: pkg/tsdb/cloudwatch/metric_find_query.go [hunk 1]
```diff
@@ -140,7 +140,7 @@ var dimensionsMap = map[string][]string{
 	"AWS/Cognito":           {"Operation", "RiskLevel", "UserPoolId"},
 	"AWS/Connect":           {"InstanceId", "MetricGroup", "Participant", "QueueName", "Stream Type", "Type of Connection"},
 	"AWS/DataSync":          {"AgentId", "TaskId"},
-	"AWS/DDoSProtection":    {"Region", "Rule", "RuleGroup", "WebACL"},
+	"AWS/DDoSProtection":    {"ACKFlood", "ChargenReflection", "DNSReflection", "GenericUDPReflection", "MemcachedReflection", "MSSQLReflection", "NetBIOSReflection", "NTPReflection", "PortMapper", "RequestFlood", "RIPReflection", "SNMPReflection", "SSDPReflection", "SYNFlood", "UDPFragment", "UDPTraffic", "UDPReflection"},
```
[the remark this answers is not in the document]
Comment (papagian) [reply pkg/tsdb/cloudwatch/metric_find_query.go:143]: Thank you for your reply.

Can you share what you get if you run the following command?
`aws cloudwatch list-metrics --namespace "DDoSProtection"`

**Comment by JuH0LEE** [comment]
@papagian  hello 
ok refer to result
```
 / aws cloudwatch list-metrics --namespace "AWS/DDoSProtection"


{
    "Metrics": [
        {
            "Namespace": "AWS/DDoSProtection",
            "MetricName": "DDoSDetected",
            "Dimensions": [
                {
                    "Name": "ResourceArn",
                    "Value": "arn:aws:cloudfront::111111111111:distribution/CLOUDFRONT-ID"
                }
            ]
        },
        {
            "Namespace": "AWS/DDoSProtection",
            "MetricName": "DDoSAttackBitsPerSecond",
            "Dimensions": [
                {
                    "Name": "ResourceArn",
                    "Value": "arn:aws:cloudfront::111111111111:distribution/CLOUDFRONT-ID"
                },
                {
                    "Name": "AttackVector",
                    "Value": "UDPTraffic"
                }
            ]
        },
        {
            "Namespace": "AWS/DDoSProtection",
            "MetricName": "DDoSAttackBitsPerSecond",
            "Dimensions": [
                {
                    "Name": "ResourceArn",
                    "Value": "arn:aws:cloudfront::111111111111:distribution/CLOUDFRONT-ID"
                },
                {
                    "Name": "AttackVector",
                    "Value": "UDPFragment"
                }
            ]
        },
        {
            "Namespace": "AWS/DDoSProtection",
            "MetricName": "DDoSAttackBitsPerSecond",
            "Dimensions": [
                {
                    "Name": "ResourceArn",
                    "Value": "arn:aws:cloudfront::111111111111:distribution/CLOUDFRONT-ID"
                },
                {
                    "Name": "AttackVector",
                    "Value": "UDSReflection"
                }
            ]
        },
        {
            "Namespace": "AWS/DDoSProtection",
            "MetricName": "DDoSAttackBitsPerSecond",
            "Dimensions": [
                {
                    "Name": "ResourceArn",
                    "Value": "arn:aws:cloudfront::111111111111:distribution/CLOUDFRONT-ID"
                },
                {
                    "Name": "AttackVector",
                    "Value": "GenericUDPReflection"
                }
            ]
        },
        {
            "Namespace": "AWS/DDoSProtection",
            "MetricName": "DDoSAttackBitsPerSecond",
            "Dimensions": [
                {
                    "Name": "ResourceArn",
                    "Value": "arn:aws:cloudfront::111111111111:distribution/CLOUDFRONT-ID"
                },
                {
                    "Name": "AttackVector",
                    "Value": "DNSReflection"
                }
            ]
        },
        {
            "Namespace": "AWS/DDoSProtection",
            "MetricName": "DDoSAttackBitsPerSecond",
            "Dimensions": [
                {
                    "Name": "ResourceArn",
                    "Value": "arn:aws:cloudfront::111111111111:distribution/CLOUDFRONT-ID"
                },
                {
                    "Name": "AttackVector",
                    "Value": "NTPReflection"
                }
            ]
        },
        {
            "Namespace": "AWS/DDoSProtection",
            "MetricName": "DDoSAttackBitsPerSecond",
            "Dimensions": [
                {
                    "Name": "ResourceArn",
                    "Value": "arn:aws:cloudfront::111111111111:distribution/CLOUDFRONT-ID"
                },
                {
                    "Name": "AttackVector",
                    "Value": "ChargenReflection"
                }
            ]
        },
        {
            "Namespace": "AWS/DDoSProtection",
            "MetricName": "DDoSAttackBitsPerSecond",
            "Dimensions": [
                {
                    "Name": "ResourceArn",
                    "Value": "arn:aws:cloudfront::111111111111:distribution/CLOUDFRONT-ID"
                },
                {
                    "Name": "AttackVector",
                    "Value": "SSDPReflection"
                }
            ]
        },
        {
            "Namespace": "AWS/DDoSProtection",
            "MetricName": "DDoSAttackBitsPerSecond",
            "Dimensions": [
                {
                    "Name": "ResourceArn",
                    "Value": "arn:aws:cloudfront::111111111111:distribution/CLOUDFRONT-ID"
                },
                {
                    "Name": "AttackVector",
                    "Value": "PortMapper"
                }
            ]
        },
        {
            "Namespace": "AWS/DDoSProtection",
            "MetricName": "DDoSAttackBitsPerSecond",
            "Dimensions": [
                {
                    "Name": "ResourceArn",
                    "Value": "arn:aws:cloudfront::111111111111:distribution/CLOUDFRONT-ID"
                },
                {
                    "Name": "AttackVector",
                    "Value": "RIPReflection"
                }
            ]
        },
        {
            "Namespace": "AWS/DDoSProtection",
            "MetricName": "DDoSAttackBitsPerSecond",
            "Dimensions": [
                {
                    "Name": "ResourceArn",
                    "Value": "arn:aws:cloudfront::111111111111:distribution/CLOUDFRONT-ID"
                },
                {
                    "Name": "AttackVector",
                    "Value": "SNMPReflection"
                }
            ]
        },
        {
            "Namespace": "AWS/DDoSProtection",
            "MetricName": "DDoSAttackBitsPerSecond",
            "Dimensions": [
                {
                    "Name": "ResourceArn",
                    "Value": "arn:aws:cloudfront::111111111111:distribution/CLOUDFRONT-ID"
                },
                {
                    "Name": "AttackVector",
                    "Value": "MSSQLReflection"
                }
            ]
        },
        {
            "Namespace": "AWS/DDoSProtection",
            "MetricName": "DDoSAttackBitsPerSecond",
            "Dimensions": [
                {
                    "Name": "ResourceArn",
                    "Value": "arn:aws:cloudfront::111111111111:distribution/CLOUDFRONT-ID"
                },
                {
                    "Name": "AttackVector",
                    "Value": "NetBIOSReflection"
                }
            ]
        },
        {
            "Namespace": "AWS/DDoSProtection",
            "MetricName": "DDoSAttackBitsPerSecond",
            "Dimensions": [
                {
                    "Name": "ResourceArn",
                    "Value": "arn:aws:cloudfront::111111111111:distribution/CLOUDFRONT-ID"
                },
                {
                    "Name": "AttackVector",
                    "Value": "MemcachedReflection"
                }
            ]
        },
        {
            "Namespace": "AWS/DDoSProtection",
            "MetricName": "DDoSAttackBitsPerSecond",
            "Dimensions": [
                {
                    "Name": "ResourceArn",
                    "Value": "arn:aws:cloudfront::111111111111:distribution/CLOUDFRONT-ID"
                },
                {
                    "Name": "AttackVector",
                    "Value": "SYNFlood"
                }
            ]
        },
        {
            "Namespace": "AWS/DDoSProtection",
            "MetricName": "DDoSAttackBitsPerSecond",
            "Dimensions": [
                {
                    "Name": "ResourceArn",
                    "Value": "arn:aws:cloudfront::111111111111:distribution/CLOUDFRONT-ID"
                },
                {
                    "Name": "AttackVector",
                    "Value": "ACKFlood"
                }
            ]
        },
        {
            "Namespace": "AWS/DDoSProtection",
            "MetricName": "DDoSAttackRequestsPerSecond",
            "Dimensions": [
                {
                    "Name": "ResourceArn",
                    "Value": "arn:aws:cloudfront::111111111111:distribution/CLOUDFRONT-ID"
                },
                {
                    "Name": "AttackVector",
                    "Value": "HTTPReflection"
                }
            ]
        },
        {
            "Namespace": "AWS/DDoSProtection",
            "MetricName": "DDoSAttackRequestsPerSecond",
            "Dimensions": [
                {
                    "Name": "ResourceArn",
                    "Value": "arn:aws:cloudfront::111111111111:distribution/CLOUDFRONT-ID"
                },
                {
                    "Name": "AttackVector",
                    "Value": "HTTPReflection"
                }
            ]
        },
        {
            "Namespace": "AWS/DDoSProtection",
            "MetricName": "DDoSDetected",
            "Dimensions": [
                {
                    "Name": "ResourceArn",
                    "Value": "arn:aws:cloudfront::111111111111:distribution/CLOUDFRONT-ID"
                }
            ]
        },
        {
            "Namespace": "AWS/DDoSProtection",
            "MetricName": "DDoSAttackBitsPerSecond",
            "Dimensions": [
                {
                    "Name": "ResourceArn",
                    "Value": "arn:aws:cloudfront::111111111111:distribution/CLOUDFRONT-ID"
                },
                {
                    "Name": "AttackVector",
                    "Value": "UDPTraffic"
                }
            ]
        },
        {
            "Namespace": "AWS/DDoSProtection",
            "MetricName": "DDoSAttackBitsPerSecond",
            "Dimensions": [
                {
                    "Name": "ResourceArn",
                    "Value": "arn:aws:cloudfront::111111111111:distribution/CLOUDFRONT-ID"
                },
                {
                    "Name": "AttackVector",
                    "Value": "UDPFragment"
                }
            ]
        },
        {
            "Namespace": "AWS/DDoSProtection",
            "MetricName": "DDoSAttackBitsPerSecond",
            "Dimensions": [
                {
                    "Name": "ResourceArn",
                    "Value": "arn:aws:cloudfront::111111111111:distribution/CLOUDFRONT-ID"
                },
                {
                    "Name": "AttackVector",
                    "Value": "UDSReflection"
                }
            ]
        },
        {
            "Namespace": "AWS/DDoSProtection",
            "MetricName": "DDoSAttackBitsPerSecond",
            "Dimensions": [
                {
                    "Name": "ResourceArn",
                    "Value": "arn:aws:cloudfront::111111111111:distribution/CLOUDFRONT-ID"
                },
                {
                    "Name": "AttackVector",
                    "Value": "GenericUDPReflection"
                }
            ]
        },
        {
            "Namespace": "AWS/DDoSProtection",
            "MetricName": "DDoSAttackBitsPerSecond",
            "Dimensions": [
                {
                    "Name": "ResourceArn",
                    "Value": "arn:aws:cloudfront::111111111111:distribution/CLOUDFRONT-ID"
                },
                {
                    "Name": "AttackVector",
                    "Value": "DNSReflection"
                }
            ]
        },
        {
            "Namespace": "AWS/DDoSProtection",
            "MetricName": "DDoSAttackBitsPerSecond",
            "Dimensions": [
                {
                    "Name": "ResourceArn",
                    "Value": "arn:aws:cloudfront::111111111111:distribution/CLOUDFRONT-ID"
                },
                {
                    "Name": "AttackVector",
                    "Value": "NTPReflection"
                }
            ]
        },
        {
            "Namespace": "AWS/DDoSProtection",
            "MetricName": "DDoSAttackBitsPerSecond",
            "Dimensions": [
                {
                    "Name": "ResourceArn",
                    "Value": "arn:aws:cloudfront::111111111111:distribution/CLOUDFRONT-ID"
                },
                {
                    "Name": "AttackVector",
                    "Value": "ChargenReflection"
                }
            ]
        },
        {
            "Namespace": "AWS/DDoSProtection",
            "MetricName": "DDoSAttackBitsPerSecond",
            "Dimensions": [
                {
                    "Name": "ResourceArn",
                    "Value": "arn:aws:cloudfront::111111111111:distribution/CLOUDFRONT-ID"
                },
                {
                    "Name": "AttackVector",
                    "Value": "SSDPReflection"
                }
            ]
        },
        {
            "Namespace": "AWS/DDoSProtection",
            "MetricName": "DDoSAttackBitsPerSecond",
            "Dimensions": [
                {
                    "Name": "ResourceArn",
                    "Value": "arn:aws:cloudfront::111111111111:distribution/CLOUDFRONT-ID"
                },
                {
                    "Name": "AttackVector",
                    "Value": "PortMapper"
                }
            ]
        },
        {
            "Namespace": "AWS/DDoSProtection",
            "MetricName": "DDoSAttackBitsPerSecond",
            "Dimensions": [
                {
                    "Name": "ResourceArn",
                    "Value": "arn:aws:cloudfront::111111111111:distribution/CLOUDFRONT-ID"
                },
                {
                    "Name": "AttackVector",
                    "Value": "RIPReflection"
                }
            ]
        },
        {
            "Namespace": "AWS/DDoSProtection",
            "MetricName": "DDoSAttackBitsPerSecond",
            "Dimensions": [
                {
                    "Name": "ResourceArn",
                    "Value": "arn:aws:cloudfront::111111111111:distribution/CLOUDFRONT-ID"
                },
                {
                    "Name": "AttackVector",
                    "Value": "SNMPReflection"
                }
            ]
        },
        {
            "Namespace": "AWS/DDoSProtection",
            "MetricName": "DDoSAttackBitsPerSecond",
            "Dimensions": [
                {
                    "Name": "ResourceArn",
                    "Value": "arn:aws:cloudfront::111111111111:distribution/CLOUDFRONT-ID"
                },
                {
                    "Name": "AttackVector",
                    "Value": "MSSQLReflection"
                }
            ]
        },
        {
            "Namespace": "AWS/DDoSProtection",
            "MetricName": "DDoSAttackBitsPerSecond",
            "Dimensions": [
                {
                    "Name": "ResourceArn",
                    "Value": "arn:aws:cloudfront::111111111111:distribution/CLOUDFRONT-ID"
                },
                {
                    "Name": "AttackVector",
                    "Value": "NetBIOSReflection"
                }
            ]
        },
        {
            "Namespace": "AWS/DDoSProtection",
            "MetricName": "DDoSAttackBitsPerSecond",
            "Dimensions": [
                {
                    "Name": "ResourceArn",
                    "Value": "arn:aws:cloudfront::111111111111:distribution/CLOUDFRONT-ID"
                },
                {
                    "Name": "AttackVector",
                    "Value": "MemcachedReflection"
                }
            ]
        },
        {
            "Namespace": "AWS/DDoSProtection",
            "MetricName": "DDoSAttackBitsPerSecond",
            "Dimensions": [
                {
                    "Name": "ResourceArn",
                    "Value": "arn:aws:cloudfront::111111111111:distribution/CLOUDFRONT-ID"
                },
                {
                    "Name": "AttackVector",
                    "Value": "SYNFlood"
                }
            ]
        },
        {
            "Namespace": "AWS/DDoSProtection",
            "MetricName": "DDoSAttackBitsPerSecond",
            "Dimensions": [
                {
                    "Name": "ResourceArn",
                    "Value": "arn:aws:cloudfront::111111111111:distribution/CLOUDFRONT-ID"
                },
                {
                    "Name": "AttackVector",
                    "Value": "ACKFlood"
                }
            ]
        },
        {
            "Namespace": "AWS/DDoSProtection",
            "MetricName": "DDoSAttackRequestsPerSecond",
            "Dimensions": [
                {
                    "Name": "ResourceArn",
                    "Value": "arn:aws:cloudfront::111111111111:distribution/CLOUDFRONT-ID"
                },
                {
                    "Name": "AttackVector",
                    "Value": "RequestFlood"
                }
            ]
        },
        {
            "Namespace": "AWS/DDoSProtection",
            "MetricName": "DDoSDetected",
            "Dimensions": [
                {
                    "Name": "ResourceArn",
                    "Value": "arn:aws:cloudfront::111111111111:distribution/CLOUDFRONT-ID"
                }
            ]
        },
        {
            "Namespace": "AWS/DDoSProtection",
            "MetricName": "DDoSAttackBitsPerSecond",
            "Dimensions": [
                {
                    "Name": "ResourceArn",
                    "Value": "arn:aws:cloudfront::111111111111:distribution/cloudfront-ID"
                },
                {
                    "Name": "AttackVector",
                    "Value": "UDPTraffic"
                }
            ]
        },
        {
            "Namespace": "AWS/DDoSProtection",
            "MetricName": "DDoSAttackBitsPerSecond",
            "Dimensions": [
                {
                    "Name": "ResourceArn",
                    "Value": "arn:aws:cloudfront::111111111111:distribution/CLOUDFRONT-ID"
                },
                {
                    "Name": "AttackVector",
                    "Value": "UDPFragment"
                }
            ]
        },
        {
            "Namespace": "AWS/DDoSProtection",
            "MetricName": "DDoSAttackBitsPerSecond",
            "Dimensions": [
                {
                    "Name": "ResourceArn",
                    "Value": "arn:aws:cloudfront::111111111111:distribution/CLOUDFRONT-ID"
                },
                {
                    "Name": "AttackVector",
                    "Value": "UDSReflection"
                }
            ]
        },
        {
            "Namespace": "AWS/DDoSProtection",
            "MetricName": "DDoSAttackBitsPerSecond",
            "Dimensions": [
                {
                    "Name": "ResourceArn",
                    "Value": "arn:aws:cloudfront::111111111111:distribution/CLOUDFRONT-ID"
                },
                {
                    "Name": "AttackVector",
                    "Value": "GenericUDPReflection"
                }
            ]
        },
        {
            "Namespace": "AWS/DDoSProtection",
            "MetricName": "DDoSAttackBitsPerSecond",
            "Dimensions": [
                {
                    "Name": "ResourceArn",
                    "Value": "arn:aws:cloudfront::111111111111:distribution/CLOUDFRONT-ID"
                },
                {
                    "Name": "AttackVector",
                    "Value": "DNSReflection"
                }
            ]
        },
        {
            "Namespace": "AWS/DDoSProtection",
            "MetricName": "DDoSAttackBitsPerSecond",
            "Dimensions": [
                {
                    "Name": "ResourceArn",
                    "Value": "arn:aws:cloudfront::111111111111:distribution/CLOUDFRONT-ID"
                },
                {
                    "Name": "AttackVector",
                    "Value": "NTPReflection"
                }
            ]
        },
        {
            "Namespace": "AWS/DDoSProtection",
            "MetricName": "DDoSAttackBitsPerSecond",
            "Dimensions": [
                {
                    "Name": "ResourceArn",
                    "Value": "arn:aws:cloudfront::111111111111:distribution/CLOUDFRONT-ID"
                },
                {
                    "Name": "AttackVector",
                    "Value": "ChargenReflection"
                }
            ]
        },
        {
            "Namespace": "AWS/DDoSProtection",
            "MetricName": "DDoSAttackBitsPerSecond",
            "Dimensions": [
                {
                    "Name": "ResourceArn",
                    "Value": "arn:aws:cloudfront::111111111111:distribution/CLOUDFRONT-ID"
                },
                {
                    "Name": "AttackVector",
                    "Value": "SSDPReflection"
                }
            ]
        },
        {
            "Namespace": "AWS/DDoSProtection",
            "MetricName": "DDoSAttackBitsPerSecond",
            "Dimensions": [
                {
                    "Name": "ResourceArn",
                    "Value": "arn:aws:cloudfront::111111111111:distribution/CLOUDFRONT-ID"
                },
                {
                    "Name": "AttackVector",
                    "Value": "PortMapper"
                }
            ]
        },
        {
            "Namespace": "AWS/DDoSProtection",
            "MetricName": "DDoSAttackBitsPerSecond",
            "Dimensions": [
                {
                    "Name": "ResourceArn",
                    "Value": "arn:aws:cloudfront::111111111111:distribution/CLOUDFRONT-ID"
                },
                {
                    "Name": "AttackVector",
                    "Value": "RIPReflection"
                }
            ]
        },
        {
            "Namespace": "AWS/DDoSProtection",
            "MetricName": "DDoSAttackBitsPerSecond",
            "Dimensions": [
                {
                    "Name": "ResourceArn",
                    "Value": "arn:aws:cloudfront::111111111111:distribution/CLOUDFRONT-ID"
                },
                {
                    "Name": "AttackVector",
                    "Value": "SNMPReflection"
                }
            ]
        },
        {
            "Namespace": "AWS/DDoSProtection",
            "MetricName": "DDoSAttackBitsPerSecond",
            "Dimensions": [
                {
                    "Name": "ResourceArn",
                    "Value": "arn:aws:cloudfront::111111111111:distribution/CLOUDFRONT-ID"
                },
                {
                    "Name": "AttackVector",
                    "Value": "MSSQLReflection"
                }
            ]
        },
        {
            "Namespace": "AWS/DDoSProtection",
            "MetricName": "DDoSAttackBitsPerSecond",
            "Dimensions": [
                {
                    "Name": "ResourceArn",
                    "Value": "arn:aws:cloudfront::111111111111:distribution/CLOUDFRONT-ID"
                },
                {
                    "Name": "AttackVector",
                    "Value": "NetBIOSReflection"
                }
            ]
        },
        {
            "Namespace": "AWS/DDoSProtection",
            "MetricName": "DDoSAttackBitsPerSecond",
            "Dimensions": [
                {
                    "Name": "ResourceArn",
                    "Value": "arn:aws:cloudfront::111111111111:distribution/CLOUDFRONT-ID"
                },
                {
                    "Name": "AttackVector",
                    "Value": "MemcachedReflection"
                }
            ]
        },
        {
            "Namespace": "AWS/DDoSProtection",
            "MetricName": "DDoSAttackBitsPerSecond",
            "Dimensions": [
                {
                    "Name": "ResourceArn",
                    "Value": "arn:aws:cloudfront::111111111111:distribution/CLOUDFRONT-ID"
                },
                {
                    "Name": "AttackVector",
                    "Value": "SYNFlood"
                }
            ]
        },
        {
            "Namespace": "AWS/DDoSProtection",
            "MetricName": "DDoSAttackBitsPerSecond",
            "Dimensions": [
                {
                    "Name": "ResourceArn",
                    "Value": "arn:aws:cloudfront::111111111111:distribution/CLOUDFRONT-ID"
                },
                {
                    "Name": "AttackVector",
                    "Value": "ACKFlood"
                }
            ]
        },
        {
            "Namespace": "AWS/DDoSProtection",
            "MetricName": "DDoSAttackRequestsPerSecond",
            "Dimensions": [
                {
                    "Name": "ResourceArn",
                    "Value": "arn:aws:cloudfront::111111111111:distribution/CLOUDFRONT-ID"
                },
                {
                    "Name": "AttackVector",
                    "Value": "RequestFlood"
                }
            ]
        },
        {
            "Namespace": "AWS/DDoSProtection",
            "MetricName": "DDoSAttackRequestsPerSecond",
            "Dimensions": [
                {
                    "Name": "ResourceArn",
                    "Value": "arn:aws:cloudfront::111111111111:distribution/CLOUDFRONT-ID"
                },
                {
                    "Name": "AttackVector",
                    "Value": "HTTPReflection"
                }
            ]
        },
        {
            "Namespace": "AWS/DDoSProtection",
            "MetricName": "DDoSAttackRequestsPerSecond",
            "Dimensions": [
                {
                    "Name": "ResourceArn",
                    "Value": "arn:aws:cloudfront::111111111111:distribution/CLOUDFRONT-ID"
                },
                {
                    "Name": "AttackVector",
                    "Value": "RequestFlood"
                }
            ]
        }
    ]
}
```

**Pull request merged**